lib/standard/collection/abstract_collection.nit

```diff
@@ -66,6 +66,14 @@ interface Collection[E]
 	#     assert [1..1[.is_empty   == true
 	fun is_empty: Bool do return length == 0
 
+	# Alias for `not is_empty`.
```

Comment:
It's worth it! I use more `not ...is_empty` than the positive. A quick git grep in WBTW revealed that it uses the positive only 24 times, vs 44 times for the `not ...is empty`.